docs/API/V1/server_config.md

```diff
@@ -200,54 +233,29 @@ The operating status of the server.
 * `enabled`: Normal operation.
 * `disabled`: Most server API endpoints will fail with the **503** (service
 unavailable) HTTP status. However a few endpoints are always allowed:
+  * [endpoints](./endpoints.md) for server configuration information;
   * [login](./login.md) because only an authenticated user with `ADMIN` role
     can modify server configuration settings;
   * [logout](./logout.md) for consistency;
-  * [endpoints](./endpoints.md) for server configuration information;
   * [server_configuration](./server_config.md) to allow re-enabling the server
     or modifying the banner.
 * `readonly`: The server will respond to `GET` requests for information, but will return **503** (service unavailable) for any attempt to modify server state. (With the same exceptions as listed above for `disabled`.)
 
 **`message`** \
 A message to explain downtime. This is required when the `status` is `disabled`
-or `readonly` and optional otherwise. Note that a `message` when the `status`
-is `enabled` won't normally be seen, unless a client asks for the `server-state`
-configuration setting. The `server-banner` configuration setting is generally
-more appropriate under normal operating conditions.
-
-When the `status` is `disabled` or `readonly`, and an API call fails with **503**
-(service unavailable), the full JSON object will be returned as the
-`application/json` error response body. A `message` key in an error response is
-a standard convention, and many clients will display this as an explanation for
-the failure. The client also has access to any additional information provided
-in the `server-state` JSON object including the `status`.
-
-## Response body
-
-The `application/json` response body for `PUT` is exactly the same as for
-[`GET`](#response-body) when the same server configuration settings are
-requested, showing only the server configuration settings that were changed
-in the `PUT`.
-
-```
-PUT /api/v1/server/configuration/dataset-lifetime?value=4
-
-{
-    "dataset-lifetime": "4"
-}
-```
-
-or 
-
-```
-PUT /api/v1/server/configuration
-{
-    "dataset-lifetime": "4 days",
-    "server-state": {"status": "enabled"}
-}
-
-{
-    "dataset-lifetime": "4",
-    "server-state": {"status": "enabled"}
-}
-```
+or `readonly` and optional otherwise.
+
+When the server status is `disabled`, or when it's `readonly` and a client
+tries to modify some data through the API, the server will fail the request
+with a `503` (service unavailable) error, and return the full `server-state`
```

Comment:
You really have it out for comma phrases. It really all depends on which rule books you read. I usually end up trying to reformat the entire sentence to avoid the issue, but, sometimes, I just tire of that effort, and develop a strong urge to randomly scatter commas everywhere ...

Reply:
In the latter part of my high school education, omitting the comma between two independent clauses, or joining them with a comma instead of a conjunction, would result in a failing grade for the entire paper, and omitting the subject or verb from what would otherwise be a stand-alone clause was a similar infraction.  Thus, including a comma like the one here renders what is after it a "sentence fragment", and that is something up with which we shall not put (to misquote Mr. Churchill).

The above may be remedied either by omitting the comma or buy supplying the missing subject for the second clause.  Otherwise, the comma separates the clause from its subject, leaving it "dangling".

Reply:
Like I said, it depends on which style guide you follow, and the fact that your teacher was an unrelenting devotee of a particular style doesn't mean there aren't equally valid alternative styles.